src/make_watertight/Cleanup.cpp

```diff
@@ -152,7 +151,7 @@ moab::ErrorCode Cleanup::delete_small_edges(const moab::Range& surfaces, const d
         MBI()->list_entities(duplicate_edges);
       assert(1 == duplicate_edges.size());
 
-      // if the edge length is less than MERGE_TOL do nothing
+      // if the edge length is less than FACET_TOL do nothing
```

Comment:
```suggestion
      // if the edge length is less than facet_tol do nothing
```

is not this variable lower case now ?

Reply:
Ummm it should be IMO, but the line below still uses the uppercase `FACET_TOL` passed into this function.

I think it'd be a worthwhile PR correcting that variable name in the future though.

Reply:
Or maybe I'm missing the point of your comment?

Reply:
you made that change L73 (old l76), in `Arc.hpp`

I was just trying to maintain consistency... (I actually missed there were not in the same file....)

BTW I don't have a strong opinion about this....